util/network-devp2p/src/node_record.rs

```diff
@@ -0,0 +1,70 @@
+use log::*;
+use parity_crypto::publickey::Secret;
+use crate::{persistence::DiskEntity, node_table::NodeEndpoint};
+
+pub type Enr = enr::Enr<secp256k1::SecretKey>;
+
+const ENR_VERSION: &str = "v4";
+
+pub struct EnrManager {
+	secret: secp256k1::SecretKey,
+	inner: Enr,
+}
+
+#[allow(dead_code)]
```

Comment:
nit: does it make sense to move this annotation to `with_node_endpoint` and `into_enr`? Other methods are used

Reply:
moved to `cfg(test)`